eng/imports/UnitTests.targets

```diff
@@ -1,35 +1,35 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Project>
 
-  <!-- Runs unit tests for a project via xUnit. -->
+  <!-- Runs unit tests for a project via 'dotnet test'. -->
   <Target Name="Test">
     <PropertyGroup>
       <TestResultsDirectory>$(ArtifactsTestResultsDir)</TestResultsDirectory>
-      <XUnitExe>$(Pkgxunit_runner_console)\tools\net472\xunit.console.x86.exe</XUnitExe>
+      <TestRunner>dotnet test</TestRunner>
 
       <!-- For Azure DevOps to read as the test results. -->
-      <XmlTestResultsFile>$(TestResultsDirectory)$(TargetName)$(TargetExt).xml</XmlTestResultsFile>
+      <XmlTestResultsFile>$(TestResultsDirectory)$(TargetName)$(TargetExt).trx</XmlTestResultsFile>
```

Comment:
nit: It is no longer an `XmlTestResultsFile` but a `TrxTestResultsFile`.

Reply:
Fixed.